airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -113,6 +151,19 @@ public static void attemptSQLCreateAndDropTableOperations(final String outputSch
     }
   }
 
+  /**
+   * Generates a dummy AirbyteRecordMessage with random values that make no sense.
```

Comment:
Probably want to remove the `that make no sense.` part, it's not really necessary and the method makes it clear this is a dummy record

Reply:
Updated, thanks